packages/console/src/components/nav-dock/NavDock.module.css

```
@@ -0,0 +1,837 @@
+
+.dockStrip {
+  /* How far the capsule floats clear of the window's foot. One card gutter put
+     it level with the page card's own bottom edge, which read as the dock being
+     part of the footer rather than an object resting above it — the whole point
+     of a capsule instead of a bar. Two gutters is the smallest lift where the
+     page ground is legible underneath it.
+
+     The trigger line is unaffected: it stays flush at one gutter, because a
+     target the cursor can overshoot is not a target. */
+  --pf-dock-lift: calc(var(--app-card-gutter) * 2);
+  position: fixed;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)